frontend/src/api/appwriteApi.js

```diff
@@ -127,6 +127,33 @@ let api = {
 		return api.provider().database.deleteDocument(collectionId, documentId);
 	},
 
+	/* Announcement */
+	createAnnouncement: (data) => {
+		return api
+			.provider()
+			.functions.createExecution(
+				AppwriteServer.announcementFunctionID,
+				JSON.stringify({
+					"action": "addAnnouncements",
+					"announcements": [data]
+				})
+			);
+	},
```

Comment:
We agreed that we do not want to use the custom function for this. Please change this to 
```
sdk.database.createDocument('[COLLECTION_ID]', {});
```

Do not forget to add `read: ["*"]` so that every user can see the announcement